new-pipeline/dist.html

```diff
@@ -181,7 +179,7 @@ <h3><select id="selected-key4" class="multiselect" title="Selected key"></select
                                     <label><input name="build-time-toggle" type="radio" value="1"> Submission Date</label>
                                 </div>
                                 <div>
-                                    <b>Sanitize data:</b> hide data points that don't have enough submissions:
+                                    <b>sanitize-toggletize data:</b> hide data points that don't have enough submissions:
```

Comment:
I'm not sure this was an intentional change?

Reply:
Sorry, This was unintentional.